packages/deployment-server-ui/src/components/StatusTag.tsx

```diff
@@ -0,0 +1,41 @@
+import { DeployStatus, Device, DeviceType } from "deployment-server";
```

Comment:
I assume we should also refactor other places to use this component, please check